### DISCUSSION TITLE
Joomla5 click Edit Link but cannot open the form

### DISCUSSION THREAD

**Discussion by fc338339** [joomengine/Joomla-Component-Builder#967] [Extention Issues] [answered]
Dear Sir 
my user ID: fc338339@hotmail.com
my component : com_cl
Table : cl_level
Action : click Edit link but cannot go to form 
Error message : 
0 Joomla\CMS\User\UserFactory::loadUserById(): Argument #1 ($id) must be of type int, string given, called in /home/cp2/public_html/administrator/components/com_cl/src/Field/CreatedbyField.php on line 52

**Comment by Llewellynvdm** [comment, marked as answer]
Please take all issues to https://git.vdm.dev/joomla/Component-Builder/issues/new/choose
